.github/workflows/os-zoo.yml

```diff
@@ -27,17 +28,26 @@ jobs:
       image: docker.io/library/alpine:${{ matrix.tag }}
     env:
       # https://www.openwall.com/lists/musl/2022/02/16/14
-      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare' || '' }}
+      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare  -Wno-covered-switch-default' || '' }}
       CC: ${{ matrix.cc }}
     steps:
     - name: install packages
       run: apk --no-cache add build-base perl linux-headers ${{ matrix.cc }}
     - uses: actions/checkout@v4
       with:
         ref: ${{ matrix.branch }}
+    - name: Check ipv6 availability
+      run: |
+        ip -6 addr show lo | grep -q inet6
+        if [ $? -eq 0 ]
+        then
+          echo "IPV6_AVAIL=1" >> $GITHUB_ENV
+        else
+          echo "IPV6_AVAIL=0" >> $GITHUB_ENV
+        fi
     - name: config
       run: |
-        ./config --banner=Configured no-shared -Wall -Werror enable-fips --strict-warnings -DOPENSSL_USE_IPV6=0 \
+        ./config --banner=Configured no-shared -Wall -Werror enable-fips --strict-warnings -DOPENSSL_USE_IPV6=${{ env.IPV6_AVAIL }} \
```

Comment:
I doubt this override is still necessary, it looks more or less like a hack that was necessary in the past.
I'd try to remove the -DOPENSSL_USE_IPV6=x alltogether

Reply:
I agree, it seems hackish.  Are we sure that all build containers are ipv6 enabled?

Reply:
so far it seems that disabling ipv6 support here broke 100% of all test runs since about 2 weeks.
So I guess that we can live without that hack.

Reply:
ack, I've removed it entirely